liferay/headless.py

```diff
@@ -19,89 +18,99 @@ def _create_poshi_task_for(jira_local, parent_story, poshi_automation_table):
+            summary = subtask.fields.summary
+            key = subtask.key
+            test_creation_subtask_details = jira.search_issues("key =" + key)
+            assignee = test_creation_subtask_details[0].fields.assignee
+            if summary == 'Test Scenarios Coverage | Test Creation':
+                print("Updating "+key+" ...")
+                if subtask.fields.status.name == "Open":
+                    description = '*Output*' \
+                                  '\r\n # Our table with the Test scenarios/test cases to be validated in the ' \
+                                  'validation phase.' \
+                                  '\r\n # After being reviewed by the team, add a finalized table to the parent story ' \
+                                  'description' \
+                                  '\r\n # Add test cases to [Test ' \
+                                  'Map|https://docs.google.com/spreadsheets/d/19KSqxtKJQ5FHZbHKxDS3_TzptWeD0DrL-mLk0y0WFYY' \
+                                  '/edit#gid=2145200593]' \
+                                  '\r\n' \
+                                  '\r\n*Test Scenarios:*' \
+                                  '\r\n||Requirement||Test Case||Covered by unit/integration test? (Yes/No)||Test Priority (' \
+                                  'business impact)||' \
+                                  '\r\n| | | | |' \
+                                  '\r\n' \
+                                  '\r\n*Exploratory testing to consider:*' \
+                                  '\r\n||Requirement||Test Scenarios||Test Priority (business impact)||Covered by ' \
+                                  'frontend/backend Unit Test?||' \
+                                  '\r\n| | | | |'
+                    subtask.update(fields={'description': description})
+                    if assignee != 'Support QA':
+                        jira.assign_issue(subtask.id, 'support-qa')
+                        break
+                    break
+    print("Subtasks Test Creation for Headless team are up to date")
+
+
+def update_validation_subtask(jira):
+    print("Updating test validation subtasks for Headless team...")
+    stories_with_test_validation_subtask = jira.search_issues('filter=55398')
+    for story in stories_with_test_validation_subtask:
+        for subtask in story.fields.subtasks:
+            summary = subtask.fields.summary
+            key = subtask.key
+            test_validation_subtask_details = jira.search_issues("key =" + key)
+            assignee = test_validation_subtask_details[0].fields.assignee
+            if summary == 'Product QA | Test Validation - Round 1':
+                print("Updating "+key+" ...")
+                if subtask.fields.status.name == "Open":
+                    description = '\r\n*Context*' \
+                                  '\r\nExecute the tests of the parent story, and use the information in the *Test ' \
```

Comment:
same that before: It think it's better to add the \r\n at the end of the previous line so it's easier o read